README.md

```diff
@@ -41,6 +41,41 @@ As API framework, we use [Feathers](http://feathersjs.com).
     npm start
     ```
 
+## Configuration
+
+In this repository we provide you with a sample configuration file, which can be found [here](https://github.com/schulcloud/node-notification-service/blob/master/secure/config.sample.json). Copy it to `secure/config.json`, as described above.
+
+### Safari Web Notifications
+
+To send Safari Web Notifications you need an Apple developer license. Go to the [Certificates, Identifiers & Profiles](https://developer.apple.com/account/ios/certificate) section of your developer account and register a new Website Push ID. Because we are using the [apn-node](https://github.com/node-apn/node-apn) package, you are free to use the certificate file or the token. Set your configuration according to the [apn.Provider](https://github.com/node-apn/node-apn/blob/master/doc/provider.markdown) documentation in your configuration file under the `sendServices.apn` property. Here you also have to specifiy the `pushId`.
+
+Safari requires specific endpoints to register with your service. One of them has to serve the Push Package. To create it follow these steps:
+
+1. Make sure you have openssl 0.9.8 installed.
+
+2. Copy your p12-certificate to `generate-pushPackage/cert.p12`.
+
+3. Copy your icons as defined in the [documentation](https://developer.apple.com/library/content/documentation/NetworkingInternet/Conceptual/NotificationProgrammingGuideForWebsites/PushNotifications/PushNotifications.html#//apple_ref/doc/uid/TP40013225-CH3-SW5) to `generate-pushPackage/icon.iconset`.
+
+4. Setup the `generate-pushPackage/website.json` according to your needs, as defined in the [documentation](https://developer.apple.com/library/content/documentation/NetworkingInternet/Conceptual/NotificationProgrammingGuideForWebsites/PushNotifications/PushNotifications.html#//apple_ref/doc/uid/TP40013225-CH3-SW4).
```

Comment:
@maxklenk Should we ignore the authentication token, make it static or configurable? Because we don't really need it at the moment, ignoring would be the simplest solution.